packages/storage-legacy/windows/ReactNativeAsyncStorage/RNCAsyncStorage.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include "pch.h"
+#include "NativeModules.h"
+#include "DBStorage.h"
+
+using namespace winrt::Microsoft::ReactNative;
```

Comment:
We shouldn't use "using namespace" in header files as this leaks to anything that includes it.